.github/BOTMETA.yml

```
@@ -156,7 +156,7 @@ files:
     maintainers: conloos
   $inventories/nmap.py: {}
   $inventories/online.py:
-    maintainers: sieben
+    maintainers: remyleone
```

Since sieben is the author of this plugin, they need to keep being mentioned. If they shouldn't be pinged, they need to be put explicitly on the ignore list:
```suggestion
    maintainers: remyleone
    ignore: sieben
```
Same for the other places.

---

I am sieben. remyleone is my new profile :)

---

Haha ok :D In that case, you need to modify the `authors:` entries of all these plugins to mention your new GH username instead of your old one.

---

done